SomerenUI/SomerenUI.cs

```diff
@@ -104,71 +104,10 @@ private void ShowDashboardPanel()
             ShowPanel(pnlDashboard);
         }
 
-        private void ShowStudentsPanel()
+        private void FillAndDisplayListView<T>(Func<List<T>> GetData, ListView listView, Func<T, ListViewItem> CreateListViewItem)
```

Comment:
Fill and display are basically the same stuff here. We're displaying data by filling data in the listView. So, in my opinion, naming it like GetAndDisplayDataInListView() makes more sense.